src/figuresClasses.ts

```diff
@@ -2,10 +1,99 @@
+export interface Figure {
+  shape: 'triangle' | 'circle' | 'rectangle';
+  color: 'red' | 'green' | 'blue';
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  color: 'red' | 'green' | 'blue';
+
+  shape: 'triangle';
+
+  a: number;
+
+  b: number;
+
+  c: number;
+
+  constructor(
+    color: 'red' | 'green' | 'blue',
+    a: number,
+    b: number,
+    c: number,
+  ) {
+    if (a <= 0 || b <= 0 || c <= 0) {
+      throw new Error('Side must be greater than 0');
+    }
+
+    if (a + b <= c || a + c <= b || b + c <= a) {
+      throw new Error('Sides do not form a valid triangle');
+    }
+    this.color = color;
+    this.a = a;
+    this.b = b;
+    this.c = c;
+    this.shape = 'triangle';
+  }
+
+  getArea(): number {
+    const s = (this.a + this.b + this.c) / 2;
+    const area = Math.sqrt(s * (s - this.a) * (s - this.b) * (s - this.c));
+
+    return Math.floor(area * 100) / 100;
+  }
+}
+
+export class Circle implements Figure {
+  radius: number;
+
+  color: 'red' | 'green' | 'blue';
 
-export class Triangle implements Figure {}
+  shape: 'circle';
```

Comment:
The `shape` property in the `Circle` class is declared as a string literal type `'circle'`. However, it should be declared as `shape: 'circle' = 'circle';` to correctly initialize it with the value `'circle'`.